Sources/SourceKitLSP/LanguageService.swift

```diff
@@ -146,7 +157,20 @@ public protocol LanguageService: AnyObject, Sendable {
   func completion(_ req: CompletionRequest) async throws -> CompletionList
   func hover(_ req: HoverRequest) async throws -> HoverResponse?
   func symbolInfo(_ request: SymbolInfoRequest) async throws -> [SymbolDetails]
-  func openGeneratedInterface(_ request: OpenGeneratedInterfaceRequest) async throws -> GeneratedInterfaceDetails?
+
+  /// Request a generated interface of a module to display in the IDE.
+  /// **(LSP Extension)**
```

Comment:
Since this is no longer a request, we don't have to include the LSP Extension text anymore.

Reply:
Done! I hope I'll get familiar with the project asap :)